owner/src/main/java/org/aeonbits/owner/util/Reflection.java

```diff
@@ -65,4 +67,23 @@ public static Object invokeDefaultMethod(Object proxy, Method method, Object[] a
         return JAVA_8_SUPPORT.invokeDefaultMethod(proxy, method, args);
     }
 
+    public static List<Class> getAllInterfaces(Class cls) {
+        List<Class> list = new ArrayList<Class>();
+        while (cls != null) {
+            for (Class anInterface : cls.getInterfaces()) {
+                if (!list.contains(anInterface)) {
```

Comment:
I think here you should have used a `java.util.Set` instead of `java.util.List`, and you could have skipped this check.